.github/workflows/codemeta.yaml

```diff
@@ -44,3 +44,14 @@ jobs:
           name: codemeta.json
       - name: validate codemeta
         run: eossr-metadata-validator codemeta.json
+  publish:
+    needs: validate
```

Comment:
Okay, now you have the dep. Maybe do the moving of the validate step in this commit instead?